pulp_smash/tests/rpm/api_v2/test_unassociate.py

```diff
@@ -94,6 +94,50 @@ def test_02_remaining_units(self):
         }
         self.assertEqual(removed_ids & remaining_ids, set())
 
+    def test_03_last_unit_removed(self):
+        """Assert that `last_unit_removed` timestamp was not updated.
+
+        Whenever disassociate_units is called `last_unit_removed` should not
+        be updated if that cal does not remove units.
+
+        Each verification is wrapped in a subTest.
+        See :meth:`do_verify_last_unit_timestamp`.
+        """
+        type_ids = ['erratum', 'package_category', 'package_group', 'rpm']
```

Comment:
I appreciate the thoroughness, but I think this is overkill. If anything, it would be more useful to try removing a pair of made-up units where one has a bogus ID and the other has a bogus type_id. Please simplify this function and merge `do_verify_last_unit_timestamp` into this.